account_invoice_refund_option/view/view_account_invoice_refund.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="view_account_invoice_refund_form_inherit" model="ir.ui.view">
+        <field name="model">account.move.reversal</field>
+        <field name="inherit_id" ref="account.view_account_move_reversal" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date']" position="after">
+                <div colspan="4">
+                    <label for="filter_refund" string="Select product(s)" />
+                    <field name="filter_refund" />
```

Comment:
adjust the alignment of the checkbox `filter_refund`
<img width="483" height="168" alt="image" src="https://github.com/user-attachments/assets/f6a8900b-571b-41b3-80ec-23f9c1556a69" />